client/state/data-layer/wpcom-http/actions.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Internal dependencies
+ */
+import { WPCOM_HTTP_REQUEST } from 'state/action-types';
+
+/**
+ * Returns a valid WordPress.com API HTTP Request action object
+ *
+ * @param {string} apiVersion [optional]  speciific API version for request
+ * @param {Object} body [optional] JSON-serializable body for POST requests
+ * @param {string} method name of HTTP method to use
+ * @param {string} path WordPress.com API path with %s and %d placeholders, e.g. /sites/%s
+ * @param {Array} pathArgs [optional] data to replace placeholders with, in order, e.g. [ 'blarg.wordpress.com' ]
```

Comment:
Note: this is a concession to the not-yet-deployed `POST` batch endpoint. It made sense to me to include the `%s` and `%d` in the `path` because of how the batching endpoint matches that paths to the intended endpoint callbacks. We can discuss this more if requested.